openquake/engine/aelo.py

```diff
@@ -112,39 +111,40 @@ def main(lon: valid.longitude,
     This script is meant to be called from the WebUI in production mode,
     and from the command-line in testing mode.
     """
-    with mock.patch.dict(os.environ, {'OQ_APPLICATION_MODE': 'aelo'}):
-        inputs = dict(lon=lon, lat=lat, vs30=vs30, siteid=siteid)
-        if jobctx is None:
-            # in  testing mode create a new job context
-            config.directory.mosaic_dir = os.path.join(
-                os.path.dirname(CDIR), 'qa_tests_data/mosaic')
-            dic = dict(calculation_mode='custom', description='AELO')
-            [jobctx] = engine.create_jobs([dic], config.distribution.log_level,
-                                          None, getpass.getuser(), None)
+    inputs = dict(lon=lon, lat=lat, vs30=vs30, siteid=siteid)
+    if jobctx is None:
+        # in  testing mode create a new job context
+        config.directory.mosaic_dir = os.path.join(
```

Comment:
No need to override the configuration object now, just pass the `mosaic_dir` downstream